resources/data.go

```diff
@@ -104,17 +105,16 @@ func AssetDir(name string) ([]string, error) {
 }
 
 type _bintree_t struct {
-	Func func() ([]byte, error)
+	Func     func() ([]byte, error)
```

Comment:
```suggestion
	Func func() ([]byte, error)
```

Reply:
This is autogenerated code, and rather not lint it manually.